arduino-ide-extension/src/node/plotter/plotter-backend-contribution.ts

```diff
@@ -14,7 +14,8 @@ export class PlotterBackendContribution
       '..',
       '..',
       '..',
-      'build',
+      '..',
+      'node_modules',
```

Comment:
I didn't test this, but are you sure this works for the bundled version of the app as well? they have different file locations, I think we cannot include from the node_modules this way...

Reply:
No, it doesn't work. It just work locally. I'm reverting it.

Reply:
Let Node.js find the plotter app: [`require.resolve`](https://nodejs.org/api/modules.html#requireresolverequest-options).

See the working example here: https://github.com/arduino/arduino-ide/compare/fspissu/avoid-twice-serial-plotter-app...kittaakos:avoid-twice-serial-plotter-app-2.

Reply:
I was already investigating to do it in this way. Thanks for the hint @kittaakos 👍

Reply:
nice implementation, using the resolve to leverage how node looks for packages is quite clever!